.pre-commit-hooks.yaml

```
@@ -48,7 +48,7 @@
   require_serial: true
   entry: hooks/terraform_validate.sh
   language: script
-  files: (\.tf|\.tfvars)$
+  files: (\.tf|\.tfvars|\.terraform\.lock\.hcl)$
```

Not essential, though for the sake of "optimization". Up to you to accept or discard at this time.
```suggestion
  files: \.(tf(vars)?|terraform\.lock\.hcl)$
```